mobile/src/main/java/rs/readahead/washington/mobile/views/activity/onboarding/OnBoardCalculatorFragment.kt

```diff
@@ -6,29 +6,33 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
 import androidx.core.content.ContextCompat
+import androidx.core.os.bundleOf
 import com.hzontal.tella_locking_ui.CALCULATOR_ALIAS
 import org.hzontal.shared_ui.bottomsheet.BottomSheetUtils
 import rs.readahead.washington.mobile.MyApplication
 import rs.readahead.washington.mobile.R
 import rs.readahead.washington.mobile.bus.event.CamouflageAliasChangedEvent
+import rs.readahead.washington.mobile.data.sharedpref.Preferences
+import rs.readahead.washington.mobile.databinding.OnboardCalculatorFragmentBinding
+import rs.readahead.washington.mobile.databinding.OnboardLockSetFragmentBinding
 import rs.readahead.washington.mobile.util.CamouflageManager
 import rs.readahead.washington.mobile.views.base_ui.BaseFragment
 
 class OnBoardCalculatorFragment : BaseFragment() {
 
     private val cm = CamouflageManager.getInstance()
-
+    private lateinit var binding: OnboardCalculatorFragmentBinding
```

Comment:
make this intialization lazy so it would be intilized only if we are about to access it,     `private val cm by lazy {CamouflageManager.getInstance()}`